src/extension-installations/selectors.js

```diff
@@ -0,0 +1,28 @@
+import _ from 'lodash';
+import { getCollection, getOne } from '@shoutem/redux-io';
+
+export function getExtensionInstallations(state, collection = null) {
+  if (collection) {
+    return getCollection(collection, state);
+  }
+
+  const extensionInstallationsCollection = _.get(state, 'core.extensionInstallations.all');
+  if (!extensionInstallationsCollection) {
+    return null;
+  }
+
+  return getCollection(extensionInstallationsCollection, state);
+}
+
+export function getExtensionInstallation(state, one = null) {
```

Comment:
We need a better name than `one` for this argument, and variables below. Maybe `descriptor` or `installationDescriptor`.